### ISSUE TITLE
Add a button to provide the list of citations covering the items in the cohort

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#1409]
We already have citations in BQ, and we should add IDC paper to the list. We should discuss this at the next meeting.

@bcli4d @s-paquette

**Comment by fedorov** [comment]
Difficult/not possible to add citations to solr cores. Should be better to use this API perhaps: https://citation.crosscite.org/docs.html

**Comment by fedorov** [comment]
We would have a column with DOI at the level of series, and on the client get distinct DOIs and map them to citations using for example crossref API.

**Comment by bcli4d** [comment]
I get citations using the requests equivalent of:
curl -LH "Accept: text/x-bibliography; style=apa" https://doi.org/10.1126/science.169.3946.635

There are citations in both original_collections_metadata and analysis_results_metadata BQ tables.

Not sure if there is anything more needed here BQ-wise.

**Comment by s-paquette** [comment]
@bcli4d No, BQ wise nothing to add. This will either query in real-time, or we'll add a core to Solr or a table to the MySQL database with the information (still thinking that over).

**Comment by s-paquette** [comment]
So, I have a version of this on dev and test now. The main issue is in order to offer more than one citation format, we would need to make a separate call for each format for each DOI. DOI.org doesn't appear to offer a way to request more than one format at a time, or do a bulk request (though I'm still looking through their API to see if I've missed something). I have it set to MLA at the moment but we can swap that to anything we feel like. 

We cam easily add this to the other tables if you want, and to the Collections list page--where all did we want this available?

**Comment by fedorov** [comment]
Unfortunately, there was a confusion - I didn't mean to add citation to the table of series in the portal, but a button at the top level to cover what is included in the cohort. I think we should remove the button in the series table that was added.

Also, agreed to use Vancouver style, as it does not shorten the author list.

**Comment by s-paquette** [comment]
Note that this is continued in #1489
